modules/base/rendering/renderablesphereimageonline.cpp

```diff
@@ -65,6 +65,9 @@ namespace {
         );
     }
 
+    // A RenderableSphereImageOnline can be used to show an image from an online source
+    // (as a URL) on a sphere in the OpenSpace scene. For more informaiton about a general
+    // RenderableSphere see [RenderableSphere](#base_renderable_sphere).
```

Comment:
I was wondering if we should mention that the base class RenderableSphere is abstract, so that people don't think it can be instantiated from this comment. I think it is also ok to leave the mentioning about renderable sphere out since we already links to the base classes anyways.

Reply:
We decided to flatten the renderable hierarchy as it is described in the documentation.  So the user of a RenderableSphereImageOnline shouldn't know about the existence of the RenderableSphere abstract class at all.

Reply:
The best solution for now would be to do the same as here: https://github.com/OpenSpace/OpenSpace/blob/master/modules/base/rendering/renderableplaneimagelocal.cpp#L67

By adding the `RenderableSphere::Documentation()` as the second argument to the `doc` function, it inserts all of the parameters directly without any need to copy them.